packages/pds-ember/README.md

```diff
@@ -0,0 +1,33 @@
+# pds-ember
+
+Ember addon for building Structure-styled UIs.
+
+
+# Compatibility
+
+* Ember.js v3.16 or above
+* Ember CLI v2.13 or above
+* Node.js v10 or above
+
+
+# Installation
+
+```
+ember install pds-ember
+```
+
+
+# Usage
+
+[Longer description of how to use the addon in apps.]
+
+
+# Contributing
+
+See the [Contributing](CONTRIBUTING.md) guide for details.
+
+
+# License
+------------------------------------------------------------------------------
+
+This project is licensed under the [MPLv2.0 License](LICENSE.txt).
```

Comment:
`ember addon` used MIT by default, so this line was updated to reflect MPLv2 license